server/src/test/java/io/squashql/client/HttpClientQuerierTest.java

```diff
@@ -137,7 +139,7 @@ void testRunQueryWithCondition() {
     // Note. The CJ will make null appear in rows. We want to make sure null values are correctly handled.
     QueryDto query = Query
             .from("our_prices")
-            .where(SCENARIO_FIELD_NAME, Functions.eq(MAIN_SCENARIO_NAME))
```

Comment:
Could you revert this condition, test is not passing.

Reply:
My bad typo when was editing